File: specs/Qowaiv.CodeAnalysis.VisualBasic.Specs/Cases/TrojanCharactersAreNotAllowed.vb
Class Controller
    Function Access() As Boolean

        Dim access_level As String = "user"
        If (access_level <> "⁣user") Then ' Noncompliant {{Trojan Horse character U+2063 detected.}}
            '                ^
            Return True
        End If
        Return False
    End Function
End Class

File: specs/Qowaiv.CodeAnalysis.CSharp.Specs/Cases/TrojanCharactersAreNotAllowed.cs
class Controller
{
    bool Access()
    {
        string access_level = "user";
        if (access_level != "⁣user") // Noncompliant {{Trojan Horse character U+2063 detected.}}
        //                   ^
        {
            return true;
        }
        return false;
    }
}
